docs/Gemfile.lock

```diff
@@ -272,20 +272,21 @@ GEM
     unf_ext (0.0.8)
     unicode-display_width (1.8.0)
     webrick (1.7.0)
-    zeitwerk (2.5.3)
+    zeitwerk (2.5.4)
 
 PLATFORMS
   x86_64-darwin-19
   x86_64-darwin-20
+  x86_64-darwin-21
   x86_64-linux
 
 DEPENDENCIES
-  github-pages (~> 215)
+  github-pages (~> 225)
   jekyll-feed (~> 0.12)
   minima (~> 2.5)
   tzinfo (~> 1.2)
   tzinfo-data
   webrick (~> 1.7)
 
 BUNDLED WITH
-   2.2.28
+   2.3.3
```

Comment:
I'm ok with these changes being shimmed into this PR, but given that it's become quite large, I suggest we separate when possible for simpler reviews in the future.